api/diagnosis/diagnosis_routes_v2.py

```
@@ -0,0 +1,87 @@
+from api.frame.TimedQueue import TimedQueue
+from api.frame.frame_routes import uid_queues, get_or_create_queue
+
+import cv2
+import numpy as np
+from datetime import datetime, timezone
+
+router = APIRouter()
+logger = get_logger(__name__)
+
+@router.get("/diagnosis/drowsiness/v2")
+async def get_diagnosis_result_v2(request: Request, device_uid: str = Query(..., alias="deviceUid")):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_diagnosis_result_v2`

(ANN201)